From 16ae0803476aa58718df95712d1a73034345fbfb Mon Sep 17 00:00:00 2001
From: David Pacheco <dap@joyent.com>
Date: Mon, 8 Jan 2018 18:30:33 +0000
Subject: [PATCH] MANTA-3538 electric-moray: pull in node-fast#15 Reviewed by:
 Cody Peter Mello <cody.mello@joyent.com> Approved by: Cody Peter Mello
 <cody.mello@joyent.com>

---
 package.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package.json b/package.json
index 065e3bd..43a9473 100644
--- a/package.json
+++ b/package.json
@@ -16,7 +16,7 @@
         "clone": "0.1.9",
         "dtrace-provider": "0.2.8",
         "fash": "2.5.0",
-        "fast": "2.3.0",
+        "fast": "2.3.2",
         "jsprim": "2.0.0",
         "kang": "1.2.0",
         "ldapjs": "0.6.3",
-- 
2.21.0

